File: Software Development Module/Week4/Bakery.java
/*
Bakery.java
Billy Cussen
09/07/2020
*/

//This class determines how many boxes of items can be carried in a Delivery Trip for a Bakery. Each box has 6 items

public class Bakery {
    
    //Data Members
    private int items;
    private int boxes;
    private int trips;
    private final int TOTALITEMSPERBOX;
    private final int TOTALBOXESPERTRIP;

    //Constructor
    public Bakery(int items){

        this.items = items;
        this.boxes = 0;
        this.trips = 0;
        this.TOTALITEMSPERBOX = 6;
        this.TOTALBOXESPERTRIP = 2;
    }

    //Compute
    public void computeBoxesAndTrips(){

        this.boxes = (this.items/this.TOTALITEMSPERBOX);
        this.trips = (this.boxes/this.TOTALBOXESPERTRIP);
 
    }

    //Getters
    public int getBoxes(){
        return this.boxes;
    }

    public int getTrips(){
        return this.trips;
    }

}

File: Python/Software Development Module - Python/Week4/Bakery.py
"""
Bakery.py
Billy Cussen
27/09/2020
"""

class myBakery:
    def __init__(self,items):
        self.items = items
        self.boxes = 0
        self.trips = 0
        self.itemsPerBox = 6
        self.boxesPerTrip = 2

    def calculateBoxesAndTrips(self):
        self.boxes = int(self.items/self.itemsPerBox)
        self.trips = int(self.boxes/self.boxesPerTrip)